SimG4Core/Geometry/src/DDDWorld.cc

```diff
@@ -25,6 +33,44 @@ DDDWorld::DDDWorld(const DDCompactView *cpv,
   catalog = ret.sdCatalog();
 }
 
+DDDWorld::DDDWorld(const cms::DDDetector* ddd, dd4hep::sim::Geant4GeometryMaps::VolumeMap& map) {
+
+  LogVerbatim("SimG4CoreApplication") 
+    << "DD4hep_DDDWorld: initialization of DDDWorld...";
+  
+  DetElement world = ddd->description()->world();
+  printout(INFO,"SimDD4CMS",
+	   "+++ DDDWorld::DDDWorld start... %s",
+	   world.name());
+  const Detector& detector = *ddd->description();
+  Geant4Converter g4Geo(detector);
+  Geant4GeometryInfo* geometry = g4Geo.create(world).detach();
+  map = geometry->g4Volumes;
+  
+  auto it = geometry->g4Volumes.find(detector.worldVolume().ptr());
+  std::cout << "The world is " << it->first.name() << "\n";
```

Comment:
MessageLogger should be used rather than cout
(you can put all the messages into a stringstream and then just pass stringstream.str() to the MessageLogger)

Reply:
There is no need to use a `stringstream` as a MessageLogger message is not sent until the local logger guard is destroyed
```C++
edm::LogInfo("Geometry") << "The world is " << it->first.name();
```
is perfectly fine.

Reply:
In this case there is a message filled (below these lines) in a for loop. (I guess this can also be done with a lambda now, but maybe a stringstream is easier.)

Reply:
yes, these and also the DDDWorld API changes discussed and agreed with @civanch are due as a next PR.

Reply:
```C++
auto l = edm::LogInfo("Geometry");
for(...) {
    l << "some message";
}
```
would accumulate the message and only output after the loop.

Reply:
To say more: this line is not needed at the end, when DD4Hep will work, so may be changed to DEBUG.

Let us clean up in the next iteration, as I said before we will work on clean up of this.